.github/stale.yml

```diff
@@ -0,0 +1,11 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+closeComment: false
```

Comment:
```suggestion
closeComment: >
  We're closing this issue because of inactivity. Feel free to re-open it if you still need help or have questions related to it.
  Truly yours, Cloud SDK Team
```

Reply:
Same as above. Also externals cannot reopen issues.